### ISSUE TITLE
Add alt text to icons and buttons

### ISSUE THREAD

**Issue by HenrikHartiala** [HSLdevcom/jore4#1440]
This is one of the findings in the accessibility audit.
Audit link
https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit?usp=sharing

Missing alt text

- [x] (Routes) Icons ([20](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g20a50ed904e_1_9))
   - [x] Go through with Design
- [x]  (Routes) Bus icon in routes list. Fix: Add mode of transportation as alt text ([36](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g24ed0538282_0_115))
   - [ ] Requires a ticket, or mention in a pre-existing ticket(?), to remember change the hardcoded alt text from Bus to dynamically based on the vehicle. Icon is currently hardcoded as a bus.
- [x]  (Map) "Show stops" ([46](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g1e3be12a1f7_0_39))
- [x]  (Map) "Tarkasteluhetki" ([46](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g1e3be12a1f7_0_39))
- [x]  (Lines) Multiple ([75](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g2520fb25fd9_0_41))([76](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g2520fb25fd9_0_59))
 
 [>]  ~(Lines&Route Search) Pagination arrows ([89](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g253769a68f1_1_0))~ New ticket: [1670](https://github.com/HSLdevcom/jore4/issues/1670)

Iterate with design if necessary.

In an old ticket(#1273), @Leitsi wrote an idea to implement: 

> Many of the buttons in Jore4 are implemented as just icons, with no text.
> Currently most of these are missing alt-texts (or tooltips).
> Should check with Juha what should be written for each icon.
>
> One idea to implement this: pass alt-text as a parameter to IconButton-component and move Tooltip component inside it (or use native alt text) so it is always required.
>
>See the accessibility audit slides:

**Comment by ellinooraikaheimo** [comment]
Awaits for @niknii-vincit to continue with this.

**Comment by niknii-vincit** [comment]
There's still plenty of `IconButton`s missing accessible texts, e.g. in the timetable preview 

![Image](https://github.com/HSLdevcom/jore4/assets/135983800/03491a56-89d2-48e2-9061-03a85ff23920)
